hazelcast/src/main/java/com/hazelcast/internal/monitor/LocalWanPublisherStats.java

```diff
@@ -15,10 +15,10 @@
  */
 
 
-package com.hazelcast.monitor;
+package com.hazelcast.internal.monitor;
```

Comment:
WAN stats are currently exposed as public SPI but will be moved into private SPI in a subsequent PR.